components/file/types.go

```diff
@@ -23,8 +23,9 @@ import (
 
 // FileConfig wraps configuration for a file implementation
 type FileConfig struct {
-	Type     string `json:"type"`
-	Metadata json.RawMessage
+	Method   string          `json:"method"`
```

Comment:
I suppose we don't need this `method` field now?

Reply:
done